### ISSUE TITLE
Copying link, dark mode, user agent

### ISSUE THREAD

**Issue by ghost** [tobykurien/WebApps#208]
I could not copy links on search page, for example if I search anything on duckduckgo and wish to copy the link from that search result in order to paste it on new sandboxed url bar to browse, I could not.

Another request is, will you add dark theme extension for websites? Currently I am using force dark mode from developers option in my phone, but it's working insufficiently.

Another issue is, will you please add some newer user agents? All those old ones pushed me blocked on fb for sometimes.

Thank you.

**Comment by tobykurien** [comment]
Copying links is difficult because long-press has been repurposed as a way to allow you to choose how to open the link. I suggest installing an app like "Share to Clipboard" (https://f-droid.org/en/packages/com.tengu.sharetoclipboard/) so thank you can long-press and use that app to copy to the clipboard.

I can look into the dark theme as per #109 

User agents have been updated.
